### PR TITLE
Renovation: Rename Preact by Inferno

### PR DESCRIPTION
**Pull request by LazyLahtak**

### CODE REVIEW

**Review by vconst — DISMISSED** [review]

**Review by vconst — DISMISSED** [review]

File: js/renovation/inferno_wrapper/__tests__/utils/test_components/base_test_widget.ts [hunk 1]
```diff
@@ -5,12 +5,12 @@ import Component from '../../../component';
 export default class BaseTestWidget extends Component {
   lastPassedProps: any;
 
-  getLastPreactPassedProps(): any {
+  getLastInfernoPassedProps(): any {
     return this.lastPassedProps;
   }
 
-  _renderPreact(props: any): void {
+  _renderInferno(props: any): void {
```
Comment (MikeVitik) [inline js/renovation/inferno_wrapper/__tests__/utils/test_components/base_test_widget.ts]: Can we use simple _render( without Inferno or Preact?

File: js/renovation/inferno_wrapper/__tests__/utils/test_components/inferno_test_widget.tsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ export class PreactTestWidgetProps {
   },
   view,
 })
-export default class PreactTestWidget extends JSXComponent(PreactTestWidgetProps) {
+export default class InfernoTestWidget extends JSXComponent(InfernoTestWidgetProps) {
```
Comment (MikeVitik) [inline js/renovation/inferno_wrapper/__tests__/utils/test_components/inferno_test_widget.tsx]: Can we rename it in something more neutral. RenovationTestWidget, or simple TestWidget

File: js/renovation/inferno_wrapper/__tests__/utils/test_components/inferno_test_widget.tsx [hunk 3]
```diff
@@ -81,7 +81,7 @@ export default class PreactTestWidget extends JSXComponent(PreactTestWidgetProps
   }
 
   @Method()
-  getLastPreactReceivedProps(): PreactTestWidgetProps {
+  getLastInfernoReceivedProps(): InfernoTestWidgetProps {
```
Comment (MikeVitik) [inline js/renovation/inferno_wrapper/__tests__/utils/test_components/inferno_test_widget.tsx]: The same here. Can we omit Inferno?

File: js/renovation/inferno_wrapper/component.ts [hunk 4]
```diff
@@ -32,7 +32,7 @@ const insertContainer = (parentNode: Element, element: Element, index: number):
   }
 }
 
-export default class PreactWrapper extends DOMComponent {
+export default class InfernoWrapper extends DOMComponent {
```
Comment (MikeVitik) [inline js/renovation/inferno_wrapper/component.ts]: Can we rename it in something more neutral? JQueryWrapper

File: js/renovation/inferno_wrapper/component.ts [hunk 5]
```diff
@@ -42,7 +42,7 @@ export default class PreactWrapper extends DOMComponent {
     class?: string;
     [name: string]: unknown;
   };
-  _preactReplaced!: boolean;
+  _infernoReplaced!: boolean;
```
Comment (MikeVitik) [inline js/renovation/inferno_wrapper/component.ts]: Omit inferno

**Review by MikeVitik — CHANGES_REQUESTED** [review]

**Review by MikeVitik — DISMISSED** [review]

**Review by vconst — APPROVED** [review]

**Pull request merged**